beta/src/pages/learn/add-react-to-a-website.md

```diff
@@ -59,216 +59,218 @@ function LikeButton() {
   const [liked, setLiked] = React.useState(false);
 
   if (liked) {
-    return 'You liked this!';
+    return 'आपको यह पसंद आया!';
   }
 
   return React.createElement(
     'button',
     {
       onClick: () => setLiked(true),
     },
-    'Like'
+    'पसंद करें'
   );
 }
 ```
 
-### Step 4: Add your React Component to the page {/*step-4-add-your-react-component-to-the-page*/}
+### स्टेप 4: पेज पर अपना React कौम्पोनॅन्ट ऐड करें {/*step-4-add-your-react-component-to-the-page*/}
 
-Lastly, add two lines to the bottom of **like_button.js**. These two lines of code find the `<div>` you added to your HTML in the first step and then display the "Like" button React component inside of it.
+अंत में, **like_button.js** के नीचे दो पंक्तियाँ ऐड करें। कोड की ये दो पंक्तियाँ पहले स्टेप में आपके द्वारा अपने HTML में ऐडे गए `<div>` को ढूंढती हैं और फिर उसके अंदर "पसंद करें" बटन React कौम्पोनॅन्ट प्रदर्शित करती हैं।
 
 ```js
-const domContainer = document.getElementById('component-goes-here');
+const domContainer = document.getElementById('कौम्पोनॅन्ट-यहाँ-जाएगा');
 ReactDOM.render(React.createElement(LikeButton), domContainer);
 ```
 
-**Congratulations! You have just rendered your first React component to your website!**
+**बधाई हो! आपने अभी-अभी अपनी वेबसाइट पर अपना पहला React कौम्पोनॅन्ट प्रस्तुत किया है!**
 
-- [View the full example source code](https://gist.github.com/rachelnabors/c64b3aeace8a191cf5ea6fb5202e66c9)
-- [Download the full example (2KB zipped)](https://gist.github.com/rachelnabors/c64b3aeace8a191cf5ea6fb5202e66c9/archive/7b41a88cb1027c9b5d8c6aff5212ecd3d0493504.zip)
+- [पूरा उदाहरण स्रोत कोड देखें](https://gist.github.com/rachelnabors/c64b3aeace8a191cf5ea6fb5202e66c9)
+- [पूरा उदाहरण डाउनलोड करें (2KB ज़िपित)](https://gist.github.com/rachelnabors/c64b3aeace8a191cf5ea6fb5202e66c9/archive/7b41a88cb1027c9b5d8c6aff5212ecd3d0493504.zip)
```

Comment:
`zipped`  => `ज़ीपड`

how about this? Or we can leave it as is in english

Reply:
I think English will be better.